e2e/vmscript-coverage/package/vmscript.js

```diff
@@ -0,0 +1,15 @@
+function addOne(inputNumber) {
```

Comment:
add some code to this file that's not covered? Just to ensure that works as well

Reply:
IMHO, this function covered by the tail call from line #14 in the same file.
Do you expect to have a new test case for this function individually?